classylss/binding.pyx

```diff
@@ -378,7 +378,7 @@ cdef class Background:
     cdef background * ba
     cdef readonly dict data
 
-    cdef readonly double Opncdm0
+    cdef readonly double Omega0_pncdm
```

Comment:
The renaming of Omega's. What about adding the astropy names on the `Cosmology` object instead? We can probably add them as an astropy_compat delegate?